src/transforms/transform-deployment.ts

```diff
@@ -101,11 +106,28 @@ const mapState = (state: string | undefined): string => {
 	}
 };
 
+const matchesEnvironment = (environment: string, globPatterns: string[] = []): boolean => {
+	for (const glob of globPatterns) {
+		if (minimatch(environment, glob)) {
+			return true;
+		}
+	}
+	return false;
+};
+
+/**
+ * Maps a given environment name to a Jira environment name using the custom mapping defined in a RepoConfig.
+ */
+export const mapEnvironmentWithConfig = (environment: string, config: Config): string | undefined => {
+	return _.keys(config?.deployments?.environmentMapping)
+		.find(jiraEnvironmentType => matchesEnvironment(environment, config.deployments?.environmentMapping?.[jiraEnvironmentType]));
+};
```

Comment:
would it be possible to match more than one environment?

Reply:
No, the first match wins. If people have overlapping patterns, they have a different problem :)